db/migrate/20170312171020_initialize_default_contexts.rb

```diff
@@ -0,0 +1,19 @@
+class InitializeDefaultContexts < ActiveRecord::Migration[5.0]
+  class PublishingContext < ActiveRecord::Base; end
+
+  def up
+    PublishingContext.create!(name: "milk_quantity")
+    PublishingContext.create!(name: "milk_quality")
+    PublishingContext.create!(name: "movement")
+    PublishingContext.create!(name: "temperature")
+    PublishingContext.create!(name: "intake")
+    PublishingContext.create!(name: "birth")
+    PublishingContext.create!(name: "calf")
+    PublishingContext.create!(name: "noise")
+    PublishingContext.create!(name: "health")
```

Comment:
I guess @drjakob would prefer a web interface. If you don't want to implement a web interface, please use a `rake task` for static data instead. I have had a lot of troubles with static data in migrations already